src/pages/index.tsx

```diff
@@ -47,7 +48,7 @@ export type HomePageProps = {
   posts: SerializedPost[];
 };
 
-export type HomePageTab = "teachers" | "curriculum" | "pupils";
+export type HomePageTab = "teachers" | "curriculum" | "pupils" | "ai";
 const isHomePageTab = (u: unknown): u is HomePageTab =>
```

Comment:
Does `isHomePageTab` need to be set if the tab is `ai`?

Reply:
I think this is needed to take the user back to the AI tab when they press the back button from another page.

Reply:
Good flag, I think this persists the same state when the page reloads. Will update